### PR TITLE
add external pgvector

### PR DESCRIPTION
**Pull request by FingerLiu**

### CODE REVIEW

**Comment by BorisPolonsky** [comment]
Thanks for your contribution. Could you please change target branch to 'dev' instead since we have other on going updates to be merged as well.

**Comment by FingerLiu** [comment]
changed to dev

File: charts/dify/templates/config.tpl [hunk 1]
```diff
@@ -232,7 +242,6 @@ MILVUS_PORT: {{ .Values.externalMilvus.port | toString | quote }}
 # The milvus password.
 # MILVUS_PASSWORD: {{ .Values.externalMilvus.password | quote }}
 # The milvus tls switch.
-MILVUS_SECURE: {{ .Values.externalMilvus.useTLS | toString | quote }}
```
Comment (BorisPolonsky) [inline charts/dify/templates/config.tpl:235]: Please restore this config

File: charts/dify/templates/config.tpl [hunk 2]
```diff
@@ -220,6 +220,16 @@ QDRANT_URL: {{ .Values.externalQdrant.endpoint }}
 # The Qdrant clinet timeout setting.
 QDRANT_CLIENT_TIMEOUT: "20"
 # The DSN for Sentry error reporting. If not set, Sentry error reporting will be disabled.
+{{- else if .Values.externalPgvector.enabled}}
+
+# Milvus configuration Only available when VECTOR_STORE is `pgvector`.
```
Comment (BorisPolonsky) [inline charts/dify/templates/config.tpl:225]: Please update this comment

**Review by BorisPolonsky — CHANGES_REQUESTED** [review]

**Comment by FingerLiu** [comment]
done

**Review by BorisPolonsky — DISMISSED** [review]

**Review by BorisPolonsky — CHANGES_REQUESTED** [review]
Please restore Milvus TLS config.

https://github.com/BorisPolonsky/dify-helm/pull/79/commits/04967506a741565fc1c95f66652e5e8cba972bc9#diff-b66d04a8a753c83075f115c52ba2ba47bebe8e3205335f8ddca381903fb74397L235

**Comment by FingerLiu** [comment]
restored Milvus TLS config

**Pull request merged**